packages/flutter/lib/src/rendering/object.dart

```diff
@@ -945,6 +967,7 @@ class PipelineOwner {
   /// always returns false.
   bool get debugDoingLayout => _debugDoingLayout;
   bool _debugDoingLayout = false;
+  bool _debugDoingChildrenLayout = false;
```

Comment:
```suggestion
  bool _debugDoingChildLayout = false;
```
...because it just reads better.  I'd either say "I'm doing child layout" or "I'm laying out children", but not "I'm doing children layout".